Create on-silence_hogen-bays.md

JJM317 opened the pull request.

buddhist-uni commented on _content/articles/on-silence_hogen-bays.md:
```diff
@@ -0,0 +1,26 @@
+  - soto
+  - zen
+year: 2008
+journal: Pacific World: "Journal of the Institute of Buddhist Studies"
```
just `journal: pw`

buddhist-uni commented on _content/articles/on-silence_hogen-bays.md:
```diff
@@ -0,0 +1,26 @@
+course: sati
+tags:
+  - problems
+  - soto
```
Let's remove `soto` here

buddhist-uni commented on _content/articles/on-silence_hogen-bays.md:
```diff
@@ -0,0 +1,26 @@
+drive_links:
+  - "https://drive.google.com/file/d/1Da40Ot92Uv05LqDWKHieONqhqtkXwjlj/view?usp=sharing"
+course: sati
+tags:
```
Let's maybe add `hearing` and ...`samatha`?

JJM317 commented:
Ok, I removed Soto and added Samatha and hearing. I tagged Soto since
silence is a big part of the Soto (Caodong) school - an important term for
them is mozhao (silent [mo] illumination/awareness [zhao]. But would that
tag be too limiting sinceZen schools are directly discussed? Also, if we
put samatha, shouldn't we put vipassana? In Chan/Zen zazen (zuochan)  is
seen as simultaneous samatha and vipassana. Even if we take it as in
tandem, I think this article, speaking on Great Silence, would include a
vipassana aspect. This leads me to another thought, I'm not sure what
hearing has to do with this article. I know Bays makes a one-off about
silence behind noises, but the article seems to be more about a
metaphorical silence which is awakening. Just my two-cents in case it is
helpful.

Kindly,
~Joe

On Tue, Aug 15, 2023 at 8:52 AM The Open Buddhist University <
***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In _content/articles/on-silence_hogen-bays.md
> <https://github.com/buddhist-uni/buddhist-uni.github.io/pull/158#discussion_r1294544641>
> :
>
> > @@ -0,0 +1,26 @@
> +---
> +title: "On Silence"
> +authors:
> +  - "Hogen Bays"
> +external_url: "https://www.shin-ibs.edu/documents/pwj3-10/02Bays.pdf"
> +source_url: "https://pwj.shin-ibs.edu/pacific-world-third-series-number-10-fall-2008"
> +drive_links:
> +  - "https://drive.google.com/file/d/1Da40Ot92Uv05LqDWKHieONqhqtkXwjlj/view?usp=sharing"
> +course: sati
> +tags:
> +  - problems
> +  - soto
> +  - zen
> +year: 2008
> +journal: Pacific World: "Journal of the Institute of Buddhist Studies"
>
> just journal: pw
> ------------------------------
>
> In _content/articles/on-silence_hogen-bays.md
> <https://github.com/buddhist-uni/buddhist-uni.github.io/pull/158#discussion_r1294549664>
> :
>
> > @@ -0,0 +1,26 @@
> +---
> +title: "On Silence"
> +authors:
> +  - "Hogen Bays"
> +external_url: "https://www.shin-ibs.edu/documents/pwj3-10/02Bays.pdf"
> +source_url: "https://pwj.shin-ibs.edu/pacific-world-third-series-number-10-fall-2008"
> +drive_links:
> +  - "https://drive.google.com/file/d/1Da40Ot92Uv05LqDWKHieONqhqtkXwjlj/view?usp=sharing"
> +course: sati
> +tags:
> +  - problems
> +  - soto
>
> Let's remove soto here
> ------------------------------
>
> In _content/articles/on-silence_hogen-bays.md
> <https://github.com/buddhist-uni/buddhist-uni.github.io/pull/158#discussion_r1294551512>
> :
>
> > @@ -0,0 +1,26 @@
> +---
> +title: "On Silence"
> +authors:
> +  - "Hogen Bays"
> +external_url: "https://www.shin-ibs.edu/documents/pwj3-10/02Bays.pdf"
> +source_url: "https://pwj.shin-ibs.edu/pacific-world-third-series-number-10-fall-2008"
> +drive_links:
> +  - "https://drive.google.com/file/d/1Da40Ot92Uv05LqDWKHieONqhqtkXwjlj/view?usp=sharing"
> +course: sati
> +tags:
>
> Let's maybe add hearing and ...samatha?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/buddhist-uni/buddhist-uni.github.io/pull/158#pullrequestreview-1578482980>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7JN4IMF7CLXNZYY556TYR3XVNWJBANCNFSM6AAAAAA3QLK5J4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>

buddhist-uni commented:
My thinking was just that tagging both Zen and Soto is a bit redundant. If you think that this isn't really applicable to Rinzai, then we should switch the tag from Zen to Soto. What do you think? Does this jive with non-Soto Zen?

The hearing tag was a bit of a McGuffin... I imagine some Buddhist coming across this in the (hypothetical, future) `hearing` tag and being delighted to get more than they asked for. I like opportunities to point back to the practice... But yes, it isn't "really" about hearing... Am I being too cute again? 😅

JJM317 commented:
You had me at McGuffin - hearing should definitely stay in. And in the same
spirit, I would keep Zen over Soto. Historically, silence has been a Soto
thing (other schools even used silent illumination as a pejorative for the
Soto school), but this is the 21st and we're all cool with each other, so
let's draw everyone in on this article. Plus, I think I saw Bays
started out in Soto and switched to Rinzai. If he still values silence,
then I'm all in. Soteriology over intellectual history any day ;)

Thanks,
~Joe


--------------------

On Tue, Aug 15, 2023 at 10:20 AM The Open Buddhist University <
***@***.***> wrote:

> My thinking was just that tagging both Zen and Soto is a bit redundant. If
> you think that this isn't really applicable to Rinzai, then we should
> switch the tag from Zen to Soto. What do you think? Does this jive with
> non-Soto Zen?
>
> The hearing tag was a bit of a McGuffin... I imagine some Buddhist coming
> across this in the (hypothetical, future) hearing tag and being delighted
> to get more than they asked for. I like opportunities to point back to the
> practice... But yes, it isn't "really" about hearing... Am I being too cute
> again? 😅
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/buddhist-uni/buddhist-uni.github.io/pull/158#issuecomment-1679014381>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7JN4ILWRAJK7657GQF246LXVOARHANCNFSM6AAAAAA3QLK5J4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
